src/vm/i386/cgenx86.cpp

```diff
@@ -1730,6 +1730,30 @@ DWORD getextcpuid(DWORD arg1, DWORD arg2, unsigned char result[16])
     }
 }
 
+extern "C"
+__declspec(naked)
```

Comment:
Yes, in the case of this function, removing the "ret" should be enough. Since the cpuid and getcpuid functions above are not naked and similar to this function, I was wondering why this one needs to be naked.